Formula/r/radvd.rb

```diff
@@ -0,0 +1,74 @@
+class Radvd < Formula
+  desc "IPv6 Router Advertisement Daemon"
+  homepage "https://radvd.litech.org/"
+  url "https://github.com/radvd-project/radvd/releases/download/v2.20/radvd-2.20.tar.xz"
+  sha256 "25d2960fb977ac35c45a8d85b71db22ed8af325db7dbf4a562fb03eab2848dcd"
+  license "radvd"
+
+  livecheck do
+    url :stable
+    regex(/^v?(\d+(?:\.\d+)+)$/i)
+    strategy :github_latest
+  end
+
+  head do
+    url "https://github.com/radvd-project/radvd.git", branch: "master"
+    depends_on "autoconf" => :build
+    depends_on "automake" => :build
+  end
+
+  depends_on "pkgconf" => :build
+  depends_on "libbsd"
+  uses_from_macos "bison" => :build
+  uses_from_macos "flex" => :build
+
+  def install
+    system "./autogen.sh" if build.head?
+    system "./configure", *std_configure_args, "--disable-silent-rules"
```

Comment:
We prefer to locate standard at last.
```suggestion
    system "./configure", "--disable-silent-rules", *std_configure_args
```